devlab/config.template

```diff
@@ -113,6 +114,7 @@ ext_cidr = <dst_ext_cidr>
 user = <dst_user>
 password = <dst_password> 
 tenant = <dst_tenant>
+service_tenant = services
```

Comment:
Hi Roman,

In my understanding it should be services. That is actually why we have this issue.
In DST cloud (Fuel/MOS 6.1) the Openstack Service name is "services" not "service":
root@node-27:~# keystone tenant-get services
+-------------+-----------------------------------+
|   Property  |               Value               |
+-------------+-----------------------------------+
| description | Tenant for the openstack services |
|   enabled   |                True               |
|      id     |  d09b9514d67d4696bc2b6675b7bf6bda |
|     name    |              services             |
+-------------+-----------------------------------+